scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```
@@ -91,35 +92,38 @@ RUN set -ex; \
 	rm -rf /usr/src/libprotobuf-mutator
 
 # EVMONE
+WORKDIR cd /usr/src
```

[`WORKDIR`](https://docs.docker.com/engine/reference/builder/#workdir) takes a path, not a command. Not sure why this even works :)

```suggestion
WORKDIR /usr/src
```

---

Definitely a typo, also not sure why this worked.

---

After playing with it yesterday I noticed that this basically resulted in a dir called `'cd '` being created at FS root :)

---

That's definitely what happens, since `WOKRDIR` is basically just a `mkdir` and `cd` - what I was confused about was what happens to the 'second' directory in this case, i.e. `/usr/src`. Couldn't find anything in the official reference, so I would assume that it just gets ignored.